# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/test/browser/browser.ini
# Commit: a0732358fb87
# Full Hash: a0732358fb87ab47d7548decb6ab7d76e17491dd
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-04-04 09:29:13
# Regressor Bug: 1582318
# File Overlap Count: 1
# Description:
#   Bug 1582318: Remove shutting-down processes from pool immediately. r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69441
# ==============================================================================

diff -r a399dfff2725 -r a0732358fb87 netwerk/cookie/test/browser/browser.ini
--- a/netwerk/cookie/test/browser/browser.ini	Fri Apr 03 22:42:25 2020 +0000
+++ b/netwerk/cookie/test/browser/browser.ini	Fri Apr 03 20:32:33 2020 +0000
@@ -13,7 +13,6 @@
 [browser_storage.js]
 [browser_serviceWorker.js]
 [browser_sharedWorker.js]
-skip-if = fission #Bug 1582318
 [browser_sameSiteConsole.js]
 support-files = sameSite.sjs
 [browser_oversize.js]
